_examples/sse/webapp/client.gen.ts

```diff
@@ -549,8 +577,14 @@ export interface WebrpcStreamOptions<T> extends WebrpcOptions {
   onOpen?: () => void;
   onClose?: () => void;
 }
+
 export interface WebrpcOptions {
   headers?: HeadersInit;
   signal?: AbortSignal;
 }
 
+export interface WebrpcStreamController {
+  abort: (reason?: any) => void;
```

Comment:
can we call it `close()`, as in https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events ?

Reply:
I want to reserve the name "close" for a potential future resumable close event. What's happening here is triggering an abort event, so I think the name "abort" makes sense